frontend/src/modules/Metadata_Forms/components/fields.js

```diff
@@ -56,15 +57,11 @@ export const GlossaryTermField = (props) => {
     const response = await client.query(
       getGlossaryTree({
         nodeUri: field.glossaryNodeUri,
-        filter: { pageSize: 500 }
+        filter: { ...Defaults.filter, nodeType: 'T' }
```

Comment:
I think this should be `selectListFilter` (which has pageSize 10000) rather than `filter` (with pageSize (10) - or would need some pagination